paper-ieee-trans.tex

```diff
@@ -82,11 +82,11 @@ \section{Introduction}
 reliability of the e-Infrastructures was often compromised, affecting
 negatively their exploitation by the scientific communities.
 
-Soon it become apparent the need for a better balance between the addition of new features
+Soon it became apparent the need for a better balance between the addition of new features
 and the reliability of the released software. Consequently, a substantial part of the
-software lifecycle would have to be devoted to the quality of the software being
+software lifecycle would have to be devoted to the improvement of the quality of the software being
 delivered. At this point, evolving software engineering practices were
-considered and gradually adopted to define and implement quality procedures, the organization of teams
+considered and gradually adopted to define and implement quality procedures, the organization of development teams
```

Comment:
what about "organization of cross-functional teams" to be agile-like

Reply:
agree

Reply:
agree. 
If it's not clear we maybe we can use "cross-functional, development and operations, teams"

Reply:
fine with Cristina's suggestion

Reply:
agree. If it's not clear, we can say "cross-functional, development and operational, teams"